packages/terminal/src/node/terminal-backend-contribution.slow-spec.ts

```diff
@@ -1,18 +1,17 @@
 /*
- * Copyright (C) 2017 Ericsson and others.
+ * Copyright (C) 2018 Ericsson and others.
```

Comment:
Minor, we need to decide whether we do this or not. I have seen this in other PRs too. I would rather leave as is, or write `2017, 2018` instead.